docs/sources/reference/components/faro/faro.receiver.md

```diff
@@ -30,6 +30,14 @@ The following arguments are supported:
 Name               | Type          | Description                                  | Default | Required
 -------------------|---------------|----------------------------------------------|---------|---------
 `extra_log_labels` | `map(string)` | Extra labels to attach to emitted log lines. | `{}`    | no
+`log_format`       | `string`      | Export format for the logs                   | `logfmt`| no
+
+### Log format
+
+The following strings are recognized as valid log line formats:
+
+* `"logfmt"`: Export logs as [logfmt][] lines.
```

Comment:
The `[logfmt][]` link isn't defined anywhere. Were you thinking of linking that to https://brandur.org/logfmt ?

If yes then add `[logfmt]: https://brandur.org/logfmt` after line 40